### PR TITLE
Replace obsolete devdocs links

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) replaces links to `https://devdocs.magento.com` topics that have already been migrated to the Adobe Devsite (`https://developer.adobe.com`).

## Affected pages

- See changed files

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My changes follow the code style of this project.
- [x] I have read the **CONTRIBUTING** document.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/pages/style-guide/content/voice-and-tone.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ Most websites have eliminated personal pronouns because using pronouns like “y
 
 The tone in UI content – including navigation, tables, forms, and calls to action – is usually more focused on accuracy and conciseness, but the same standards apply: Use clear, plain language and consult global conventions to ensure content is understood by all.
 
-Consistency is critical. For example, wherever content is displayed in [Data Tables](https://devdocs.magento.com/guides/v2.4/pattern-library/displaying-data/datatable/datatable.html) or [Form Elements](https://devdocs.magento.com/guides/v2.4/pattern-library/getting-user-input/form_elements/form_elements.html), it should use standard or global conventions.
+Consistency is critical. For example, wherever content is displayed in [Data Tables](https://developer.adobe.com/commerce/admin-developer/pattern-library/displaying-data/datatable/) or [Form Elements](https://devdocs.magento.com/guidehttps://developer.adobe.com/commerce/admin-developer/pattern-library/getting-user-input/form-elements/), it should use standard or global conventions.
```
Comment (meker12) [inline src/pages/style-guide/content/voice-and-tone.md:132]:
```suggestion
Consistency is critical. For example, wherever content is displayed in [Data Tables](https://developer.adobe.com/commerce/admin-developer/pattern-library/displaying-data/datatable/) or [Form Elements](https://developer.adobe.com/commerce/admin-developer/pattern-library/getting-user-input/form-elements/), it should use standard or global conventions.
```

**Review by meker12 — APPROVED** [review]
One suggestion to fix link target.

File: src/pages/style-guide/content/voice-and-tone.md (hunk 1 above)
Comment (jeff-matthews) [reply src/pages/style-guide/content/voice-and-tone.md:132]: Sorry, I had auto-merge enabled. I'll open another PR.

Comment (jeff-matthews) [reply src/pages/style-guide/content/voice-and-tone.md:132]: @meker12, here's the PR: https://github.com/AdobeDocs/commerce-admin-developer/pull/12

**Pull request merged**